openapi.yaml

```diff
@@ -209,9 +438,324 @@ definitions:
+      attributes:
+        type: object
+        properties:
+          round:
+            $ref: '#/definitions/round'
+          playerIds:
+            $ref: '#/definitions/playerIds'
+          minimumBet:
+            $ref: '#/definitions/minimumBet'
+          maximumBet:
+            $ref: '#/definitions/maximumBet'
+          betPool:
+            $ref: '#/definitions/betPool'
+          tableCards:
+            $ref: '#/definitions/cards'
+      links:
+        $ref: '#/definitions/selfLink'
+  gameResult:
+    properties:
+      data:
+        $ref: '#/definitions/gameResource'
+  gameResults:
+    properties:
+      data:
+        type: array
+        items:
+          $ref: "#/definitions/gameResource"
+  gameType:
+    type: string
+    enum:
+      - game
+  maximumBet:
+    type: integer
+    minimum: 1
+    description: The maximum bet every player can place each round.
+    example: 2000
+  minimumBet:
+    type: integer
+    minimum: 1
+    description: The minimum bet every player have to place at least each round.
+    example: 1000
+  playerBet:
+    type: integer
+    minimum: 0
+    description: The bet from a specific player.
+  playerEmail:
+    type: string
+    example: abc@efg.com
+    format: email
+  memberExpAboveLevel:
+    type: integer
+    minimum: 0
+    description: Player's experience point above current level.
+    example: 1234
+  playerId:
+    type: string
+    example: '101'
+  playerIds:
+    type: array
+    items:
+      $ref: '#/definitions/playerId'
+  playerInGamePatchBody:
+    properties:
+      data:
+        type: object
+        properties:
+          id:
+            $ref: '#/definitions/playerId'
+          type:
+            $ref: '#/definitions/playerType'
+          attributes:
+            type: object
+            properties:
+              playerBet:
+                $ref: '#/definitions/playerBet'
+              playerStatus:
+                $ref: '#/definitions/playerStatus'
+              playerCards:
+                $ref: '#/definitions/cards'
+            additionalProperties: false
+            required:
+              - id
+              - type
+        additionalProperties: false
+    required:
+      - data
+    additionalProperties: false
+  playerInGameResource:
+    properties:
+      id:
+        $ref: '#/definitions/playerId'
+      type:
+        $ref: '#/definitions/playerType'
+      attributes:
+        type: object
+        properties:
+          memberName:
+            $ref: '#/definitions/memberName'
+          memberLevel:
+            $ref: '#/definitions/memberLevel'
+          memberExpAboveLevel:
+            $ref: '#/definitions/memberExpAboveLevel'
+          playerBet:
+            $ref: '#/definitions/playerBet'
+          playerStatus:
+            $ref: '#/definitions/playerStatus'
+          playerCards:
+            $ref: '#/definitions/cards'
+  playerInGameResult:
+    properties:
+      data:
+        $ref: '#/definitions/playerInGameResource'
```

Comment:
Here is an example player result based on your design:

```{
  "data": {
    "id": "101",
    "type": "player",
    "attributes": {
      "memberName": "Jack",
      "memberLevel": 20,
      "memberExpAboveLevel": 1234,
      "playerBet": 0,
      "playerStatus": "folded",
      "playerCards": {
        "cards": [
          {
            "cardNumber": "A",
            "CardSuit": "Diamond"
          }
        ]
      }
    }
  }
}
```

It's no need to put a `cards` array inside of `playerCards`, instead, you can directly make `playerCards` become an array. Same with `tableCards` in `gameResult`, `PATCH`, and `POST` body.